api/backend/ecs/load_balancer_manager.go

```diff
@@ -439,8 +439,10 @@ func (e *ECSLoadBalancerManager) portToListener(port models.Port) (*elb.Listener
 		containerProtocol = "TCP"
 	}
 
-	var certificateARN string
-	if port.CertificateName != "" {
+	// use cert arn if specified by the user
+	// otherwise, if name is specified, convert it to an arn
+	certificateARN := port.CertificateARN
+	if certificateARN == "" && port.CertificateName != "" {
```

Comment:
The problem I foresee comes further down the file in the `getCertificateARN` method on [line 459](https://github.com/quintilesims/layer0/pull/519/files#diff-b700b5ddecdc691815d57b5e44e798daR459) in your branch. 

The list of certificates retrieves from IAM will not match the list of certificates in ACM. If you provide a `certificateARN`, none of the certs pulled from IAM will match when iterated through.

Reply:
yeah i think we seemed to be confused about IAM versus ACM

Reply:
This actually works out well in that we don't need to do any work on our end. Certificate ARNs in both IAM and ACM place the name, or identifier (in the ACM case), after the last trailing slash of the ARN. So, our algorithm for converting certificate ARNs to it's name actually already works for ACM.

Reply:
ahhh, I see what you're saying. 

We're not using `getCertificateARN` to obtain an ACM ARN from a name in this case, we're requiring that the user specifies an ACM ARN and only an ARN. We will never return ACM certs via just their name with this changeset.

Reply:
If I'm correct, that is a slight "bug" in regards to a full ACM implementation but it would still definitely unblock affected users.

Reply:
so consumers will need to specify FULL ARN to use ACM certs, and friendly name to use IAM certs?

Reply:
@andycmaj I believe IAM can use either friendly name or ARN, but ACM must use the ARN.